Misc/Copy-ItemToBcContainer.ps1

```diff
@@ -0,0 +1,66 @@
+﻿<# 
+ .Synopsis
+  Copy an item (file or folder) from the host system to a container.
+ .Description
+  Copy an item (file or folder) from the host system to a container.
+ .Parameter containerName
+  Name of the container to which you want to copy the item.
+ .Parameter localPath
+  Path to the item on the host system which has to be copied.
+ .Parameter containerPath
+  Path in the container where the item will be placed.
+  If the source item is a folder, 'containerPath' will be created (or treated) as a folder and all items inside the source folder will be copied to it.
+ .Example
+  Copy-ItemToBcContainer -containerName test2 -localPath "c:\temp\myfile.txt" -containerPath "c:\run\my\myfile.txt"
+#>
+function Copy-ItemToBcContainer {
+    Param (
+        [string] $containerName = $bcContainerHelperConfig.defaultContainerName,
+        [Parameter(Mandatory = $true)]
+        [string] $localPath,
+        [Parameter(Mandatory = $false)]
+        [string] $containerPath = $localPath
```

Comment:
when copying into the container, I think it is OK to default containerPath - you cannot really overwrite anything dangerous.

Reply:
Yeah, just stating that you don't need to make containerPath mandatory here.
The only reason for doing this on the other command would be that the function could mess up your local machine pretty bad by transferring one wrong parameter.